src/main/kotlin/no/nav/dagpenger/journalføring/arena/JournalføringArena.kt

```diff
@@ -54,41 +67,63 @@ class JournalføringArena(val env: Environment, val oppslagHttpClient: OppslagHt
         )
     }
 
-    private fun filterJournalpostTypes(journalpostType: JournalpostType): Boolean {
-        return when (journalpostType) {
-            NY, GJENOPPTAK, ETTERSENDING -> true
-            UKJENT, MANUELL -> false
-        }
+    fun shouldBeProcessed(behov: Behov): Boolean {
```

Comment:
It's easier to test this if it's a separate function outside of JournalføringArena

Reply:
Good point! Fixed.